tests/Functional/Execution/ValuesHelperTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Digia\GraphQL\Test\Functional\Execution;
+
+use Digia\GraphQL\Execution\ExecutionContext;
+use Digia\GraphQL\Language\Node\ArgumentsAwareInterface;
+use Digia\GraphQL\Language\Node\OperationDefinitionNode;
+use Digia\GraphQL\Test\TestCase;
+use function Digia\GraphQL\Execution\coerceArgumentValues;
+use function Digia\GraphQL\Execution\coerceVariableValues;
+use function Digia\GraphQL\parse;
+use function Digia\GraphQL\Type\booleanType;
+use function Digia\GraphQL\Type\newNonNull;
+use function Digia\GraphQL\Type\newObjectType;
+use function Digia\GraphQL\Type\newSchema;
+use function Digia\GraphQL\Type\stringType;
+
+class ValuesHelperTest extends TestCase
+{
+    /**
+     * @throws \Digia\GraphQL\Error\InvariantException
+     * @throws \Digia\GraphQL\Error\SyntaxErrorException
+     */
+    public function testCoerceArgumentValues()
+    {
+        $schema = newSchema([
+            'query' =>
+                newObjectType([
+                    'name'   => 'Greeting',
+                    'fields' => [
+                        'greeting' => [
+                            'type' => stringType(),
+                            'args' => [
+                                'name' => [
+                                    'type' => stringType(),
+                                ]
+                            ]
+                        ]
+                    ]
+                ])
+        ]);
+
+        $documentNode = parse('query Hello($name: String) { Greeting(name: $name) }');
```

Comment:
Ah I missed this query last time, I would use line-breaks here too.

Reply:
Like earlier, this was already like this, GitHub's diff just doesn't understand that this is existing code